bblfsh-server/values.yaml

```diff
@@ -0,0 +1,26 @@
+# Default values for bblfsh-server.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: bblfsh/server
+  tag: latest
```

Comment:
* I can tie this to a version, that looks better
* Standard practice seems to be to have versions in the charts so that you only have to provide infra-related data